### PR TITLE
Add and rename fields

### PR DESCRIPTION
**Pull request by AngusCollett**

### CODE REVIEW

File: schemas/ispyb/stored_programs/sp_retrieve_dc.sql [hunk 1]
```diff
@@ -11,13 +11,15 @@ BEGIN
...
+					dc.BLSAMPLEID "blSampleId",
 					dc.blSubSampleId "blSubSampleId",
 					dc.dataCollectionNumber "dcNumber",
 					dc.APERTUREID "apertureId",
 					dc.POSITIONID "positionId",
+					dc.screeningOrigId "screeningOrigId",
 					dc.startTime "startTime",
 					dc.endTime "endTime",
-					dc.runStatus "runStatus",
+					dc.runStatus "status",
```
Comment (KarlLevik) [inline schemas/ispyb/stored_programs/sp_retrieve_dc.sql:22]: Sorry, we can't rename this alias.

File: schemas/ispyb/stored_programs/sp_retrieve_dc.sql [hunk 2]
```diff
@@ -77,13 +83,15 @@ BEGIN
...
```
Comment (KarlLevik) [inline schemas/ispyb/stored_programs/sp_retrieve_dc.sql:94]: Sorry, we can't rename this alias.

File: schemas/ispyb/stored_programs/sp_retrieve_dc.sql [hunk 3]
```diff
@@ -59,6 +62,9 @@ BEGIN
 					dc.beamSizeAtSampleY "beamSizeAtSampleY",
 					dc.focalSpotSizeAtSampleX "focalSpotSizeAtSampleX",
 					dc.focalSpotSizeAtSampleY "focalSpotSizeAtSampleY",
+					dc.POLARISATION "polarisation",
+					dc.actualCenteringPosition "actualCentringPosition",
```
Comment (KarlLevik) [inline schemas/ispyb/stored_programs/sp_retrieve_dc.sql:66]: You've spelled it "actualCenteringPosition" in the gda-ispyb-api!

File: schemas/ispyb/stored_programs/sp_retrieve_dc.sql [hunk 4]
```diff
@@ -125,6 +134,9 @@ BEGIN
...
```
Comment (KarlLevik) [inline schemas/ispyb/stored_programs/sp_retrieve_dc.sql:138]: Same comment re: spelling.

**Review by KarlLevik — CHANGES_REQUESTED** [review]
Basically two issues, see below.

File: schemas/ispyb/stored_programs/sp_retrieve_dc.sql (hunk 1 above)
Comment (AngusCollett) [reply schemas/ispyb/stored_programs/sp_retrieve_dc.sql:22]: Yeah I think it is supposed to be `status` according to the master branch. I don't know why it went back to `runStatus` on my end but this is where I'm correcting it.

File: schemas/ispyb/stored_programs/sp_retrieve_dc.sql (hunk 3 above)
Comment (AngusCollett) [reply schemas/ispyb/stored_programs/sp_retrieve_dc.sql:66]: I'll change it on the api side. "Centring" is apparently the preferred English spelling but I only found this out the other day. I've seen it spelled both ways in various places.

File: schemas/ispyb/stored_programs/sp_retrieve_dc.sql (hunk 1 above)
Comment (KarlLevik) [reply schemas/ispyb/stored_programs/sp_retrieve_dc.sql:22]: Ah, so it was status, then accidentally got changed. I see. Then it's fine.

File: schemas/ispyb/stored_programs/sp_retrieve_dc.sql (hunk 3 above)
Comment (KarlLevik) [reply schemas/ispyb/stored_programs/sp_retrieve_dc.sql:66]: OK, sounds good.

**Review by KarlLevik — APPROVED** [review]

**Pull request merged**